tester.go

```diff
@@ -18,7 +20,20 @@ type Tester struct {
 
 // newTester creates a Tester based on the TesterDefinition provided
 func newTester(env map[string]string, definition tester_definition.TesterDefinition) (Tester, error) {
-	context, err := tester_context.GetTesterContext(env, definition.ExecutableFileName)
+	submissionDir, ok := env["CODECRAFTERS_SUBMISSION_DIR"]
+	if !ok {
+		return Tester{}, fmt.Errorf("CODECRAFTERS_SUBMISSION_DIR env var not found")
+	}
+
+	// check for your_program.sh
+	_, err := os.Stat(path.Join(submissionDir, definition.ExecutableFileName))
+	var context tester_context.TesterContext
+	if os.IsExist(err) {
+		context, err = tester_context.GetTesterContext(env, definition.ExecutableFileName)
+	} else {
+		context, err = tester_context.GetTesterContext(env, definition.LegacyExecutableFileName)
```

Comment:
If a file named legacyExecutableFileName isn't present on disk either, we should default to `ExecutableFileName`. Imagine a future where we're using `your_program.sh`. If the user accidentally deletes the file, we want error messages to say "your_program.sh not found", not "your_redis_server.sh not found".

Reply:
Aha, good point💡